packages/lib-user/src/components/GroupStats/GroupStats.js

```diff
@@ -159,19 +161,36 @@ function GroupStats({
         />
         {showTopContributors ? (
           <Grid
-            columns='1/2'
+            columns={size === 'large' ? ['1fr 1fr'] : ['1fr']}
             gap='30px'
           >
-            <TopContributors
-              groupId={group?.id}
-              stats={stats}
-              topContributors={topContributors}
-            />
-            <TopProjects
-              allProjectsStats={allProjectsStats}
-              grid={true}
-              projects={projects}
-            />
+            {size === 'large' ? (
+              <>
+                <TopContributors
+                  groupId={group?.id}
+                  stats={stats}
+                  topContributors={topContributors}
+                />
+                <TopProjects
+                  allProjectsStats={allProjectsStats}
+                  grid={true}
+                  projects={projects}
+                />
+              </>
+            ) : (
+              <>
+                <TopProjects
+                  allProjectsStats={allProjectsStats}
+                  grid={false}
+                  projects={projects}
+                />
+                <TopContributors
+                  groupId={group?.id}
+                  stats={stats}
+                  topContributors={topContributors}
+                />
+              </>
+            )}
```

Comment:
[minor][non-blocking] Instead of rendering the components in reverse, I'm wondering if there's a _simpler_ way to specify when one component should appear before the other.

For example, the [CSS `order` property](https://developer.mozilla.org/en-US/docs/Web/CSS/order) lets us do something like `<div> <span style="order: 0">Second</span> <span style="order: -1">First</span> </div>` to reverse the order of elements.

Alternatively, the Grid could use _named areas,_ which the TopProjects and TopContributors elements would slot into.

But saying that, I realise that those solutions might require passing variables down into the TopProjects and TopContributors components so they can handle the CSS, so those changes may not make the code more readable in the end. 🤷

Reply:
Yes it is repetitive 🤔 , though in one condition TopProjects grid is `true`, the other `false`, coupled with the initial check on `showContributors`, I'm not sure there's an alternative that clarifies the 3 different views (showContributors & large screen, showContributors & not large screen, and don't show contributors) significantly better. I think I'll leave as is for now, but will certainly stew on.